README.md

```diff
@@ -0,0 +1,38 @@
+[Discordia]: https://discordia.me
+[Discord]: https://discord.com
+
+[Easter Eggs]: https://discordia.me/en/easter-eggs
+[User Settings]: https://discordia.me/en/user-settings
+
+[Wiki.js v2]: https://wiki.js.org
+[guide]: https://docs.requarks.io/en/editors/markdown
+
+[server]: https://discord.gg/ZRJ9Ghh
+
+[license]: https://github.com/Discordia-Development/wiki/blob/new/license.md
+
+# Discordia
+[Discordia]( is of if not the largest Wiki all about [Discord].
+
+From [Easter Eggs] to [User Settings] do we cover (almost) all importand information you may want to know about Discord.
+
+## Wanna help?
+Great! We appreciate any help to expand and improve the wiki.  
+You can contribute to this wiki in two ways.
+
+### Markdown Editor
+We use [Wiki.js v2] for rendering and handling the wiki pages on our side and recommend you to use their Markdown Editor for editing and creating pages.  
+Please take a look at their [User Guide][guide] to get a basic understanding of what the editor may offer.
+
+### Pull Request
+You can also create a Pull request here on GitHub to update and/or add pages of the wiki.  
+Please make sure that your PR targets the `new` branch, as it is the main one used by the wiki itself.
+
+## Questions?
+If you have any questions about us or the wiki, don't hesitate to join our [Discord Server][server] for help.
+
+## Licenses
+If not stated otherwhise are the contents displayed on the wiki licensed under [**Creative Commons Attribution Share Alike 4.0 International**][license].  
```

Comment:
```suggestion
Unless stated otherwise, the content displayed on the wiki are licensed under the [**Creative Commons Attribution Share Alike 4.0 International**][license].  
```